modules/account-lib/package.json

```diff
@@ -46,7 +46,7 @@
     "@types/lodash": "^4.14.151",
     "bignumber.js": "^9.0.0",
     "bs58check": "^2.1.2",
-    "casper-client-sdk": "1.0.29",
+    "casper-client-sdk": "1.0.39",
```

Comment:
Please use `^1.0.39`

Reply:
Also please commit updated `yarn.lock`

Reply:
Hi @tylerlevine , we decided stop using `^` some time ago, at least for `casper-client-sdk`. Casperabs release new versions twice a week and we had a lot of issues because of bugs on these updates. Also, each release require some sort of integration testing to make sure everything was still working as expected; which made as lose a lot of time too.
@CapnMigraine , please correct me if I'm missing something.

Regarding the `yarn.lock`, I'll update this PR with the updated one.